tests/components/roku/test_media_player.py

```diff
@@ -128,6 +128,7 @@ async def test_idle_setup(
     await setup_integration(hass, aioclient_mock, power=False)
 
     state = hass.states.get(MAIN_ENTITY_ID)
+    assert state
```

Comment:
this is due to noticing a sensor error blowing up this test (complaining about acceasing NoneType) during one of test runs. fixed that sensor issue but figured was small enough to leave this assert in.